osm2gtfs/creators/schedule_creator.py

```diff
@@ -0,0 +1,53 @@
+# coding=utf-8
+
+import sys
+import json
+
+
+class ScheduleCreator(object):
+    """
+    The ScheduleCreator loads, validates and prepares configuration data from
+    the schedule source file for further use in the script.
+
+    """
```

Comment:
Maybe you could at the URL to the documentation here?

Reply:
Added